src/views/GovernanceTaskList/index.tsx

```diff
@@ -79,22 +150,25 @@ const GovernanceTaskList = () => {
                 description="Tell the Governance Admin Team about your idea. This step lets CMS build
               context about your request and start preparing for discussions with your team."
                 status={intakeStatus}
-                link={intakeLink}
-              />
+              >
+                {intakeLink()}
```

Comment:
Having a function call as a react child is a bit odd. Could you make this it's own tiny component with props, etc?